addons/project_mrp/models/mrp_production.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class MrpProduction(models.Model):
+    _inherit = 'mrp.production'
+
+    task_id = fields.Many2one('project.task', string='Task', readonly=True)
```

Comment:
```suggestion
    task_id = fields.Many2one('project.task', string='Task', readonly=True, export_string_translation=False)
```
Since we don't use the field in a view